### ISSUE TITLE
Doesn't work out of the box with Laravel 5.1

### ISSUE THREAD

**Issue by djaiss** [wildbit/laravel-postmark-provider#3]
After doing a `composer require wildbit/laravel-postmark-provider`, we get

``` shell
Using version ^1.0 for wildbit/laravel-postmark-provider
./composer.json has been updated
> php artisan clear-compiled
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Can only install one of: guzzlehttp/guzzle[6.1.0, 5.3.0].
    - Can only install one of: guzzlehttp/guzzle[5.3.0, 6.1.0].
    - Can only install one of: guzzlehttp/guzzle[5.3.0, 6.1.0].
    - wildbit/swiftmailer-postmark 1.1.0 requires guzzlehttp/guzzle ~5.2 -> satisfiable by guzzlehttp/guzzle[5.3.0].
    - wildbit/laravel-postmark-provider 1.0.0 requires wildbit/swiftmailer-postmark ~1.1 -> satisfiable by wildbit/swiftmailer-postmark[1.1.0].
    - Installation request for wildbit/laravel-postmark-provider ^1.0 -> satisfiable by wildbit/laravel-postmark-provider[1.0.0].
    - Installation request for guzzlehttp/guzzle == 6.1.0.0 -> satisfiable by guzzlehttp/guzzle[6.1.0].


Installation failed, reverting ./composer.json to its original content.
```

My `composer.json` looks like this:

``` json
"require": {
        "php": ">=5.5.9",
        "laravel/framework": "5.1.*",
        "laravel/socialite": "^2.0",
        "guzzlehttp/guzzle": "~5.3|~6.0"
    },
```

What can I do to use the official Laravel client? I know I can use SMTP to send my emails via Postmark, but it's too bad it doesn't work with the official Laravel client.

**Comment by atheken** [comment]
Hi @djaiss, I will look at updating the swiftmailer package so that it uses the updated version of guzzle, this should resolve the laravel requirements.

**Comment by atheken** [comment]
@djaiss - Can you do me a favor and run `composer require wildbit/laravel-postmark-provider dev-master` to confirm that you can install the updated version and that mail sending works as you expect it to? Once I get confirmation, I will label a new release and you should be good to go.

**Comment by djaiss** [comment]
@atheken works like a charm.

``` shell
./composer.json has been updated
> php artisan clear-compiled
Loading composer repositories with package information
Updating dependencies (including require-dev)
  - Installing wildbit/swiftmailer-postmark (2.0.0)
    Downloading: 100%

  - Installing wildbit/laravel-postmark-provider (dev-master 79a7e8b)
    Cloning 79a7e8bde66b2bd6f314829b00ee08616847ebc5

Writing lock file
Generating autoload files
> php artisan optimize
Generating optimized class loader
```

**Comment by atheken** [comment]
This has been released as 2.0.0 -- no functional changes, but since it's not "backwards compatible," we bump the major version.
